### DISCUSSION TITLE
Azure Monitor: add Basics Logs API

### DISCUSSION THREAD

**Discussion by tkubica12** [grafana/grafana#61019] [Data Sources]
Azure Monitor data source supports classic API (today called Analytics Logs), but Azure introduced much cheaper option when user need just filtering with no advanced analytics - Basic Logs with limited set of KQL capabilities. Unfortunately, this API endpoint is different. https://learn.microsoft.com/en-us/azure/azure-monitor/logs/basic-logs-query?tabs=api-1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82913. Feel free to continue the discussion around this there. Thank you!
